CHANGELOG.md

```diff
@@ -3,7 +3,18 @@ All notable changes to this project 1.4.x per each release will be documented in
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [v1.4.0.2] - 2019.09.26
+## [v1.4.0.3] - 2019-10-03
```

Comment:
@kdanW better not to set the release date as of now, since this is a tentatively set date.

Reply:
Fixed